floodviz/hydrograph_utils.py

```diff
@@ -37,8 +41,21 @@ def parse_hydrodata(jdata):
                 dt = datetime.strptime(date + ' ' + t, '%Y-%m-%d %H:%M:%S')
                 # Convert to milliseconds for use with d3 x axis format
                 dt_ms = time.mktime(dt.timetuple()) * 1000
+                # handle missing data
+                if prev_date_ms:
```

Comment:
It might help to have more detailed comments to explain variables and steps of the calculation.